packages/fxa-support-panel/lib/api.ts

```diff
@@ -258,7 +257,8 @@ export function init(
       options: {
         handler: supportController.displayUser,
         validate: {
-          query: (queryValidator as unknown) as hapiJoi.ObjectSchema,
```

Comment:
the newer version of typescript didn't like this an I couldn't figure out how to fix it

Reply:
typescript 3.9 also has some issues with other libs. we won't be able to use it for awhile.

Reply:
I was able to get everything to 3.8.3 but 3.9 broke a couple of our 3rd party deps.